build/configs/rtl8730e/loadable_apps/defconfig

```diff
@@ -1082,7 +1086,7 @@ CONFIG_PM_DEVNAME_LEN=32
 CONFIG_PM_SLICEMS=100
 CONFIG_PM_NDOMAINS=1
 CONFIG_PM_MEMORY=2
-# CONFIG_PM_DVFS is not set
+CONFIG_PM_DVFS=y
```

Comment:
@ritesh55555 Is it ok to enable this now?

Reply:
It will only enable an api for now. No other effect on working of the system.